pkg/sbom/cyclonedx/unmarshal_test.go

```diff
@@ -308,25 +358,6 @@ func TestUnmarshaler_Unmarshal(t *testing.T) {
 									BOMRef: "pkg:k8s/k8s.io%2Fkubelet@1.27.4",
 								},
 							},
-							{
-								ID:      "k8s.io/kubernetes@1.27.4",
-								Name:    "k8s.io/kubernetes",
-								Version: "1.27.4",
-								Identifier: ftypes.PkgIdentifier{
-									PURL: &packageurl.PackageURL{
-										Type:    purl.TypeK8s,
-										Name:    "k8s.io/kubernetes",
-										Version: "1.27.4",
-									},
-									BOMRef: "pkg:k8s/k8s.io%2Fkubernetes@1.27.4",
-								},
-								DependsOn: []string{
-									"k8s.io/apiserver@1.27.4",
-									"k8s.io/controller-manager@1.27.4",
-									"k8s.io/kube-proxy@1.27.4",
-									"k8s.io/kube-scheduler@1.27.4",
-								},
```

Comment:
Trivy doesn't support dependencies between applications.
And since we have separated the applications, these dependencies will be removed